src/git.js

```diff
@@ -304,7 +326,7 @@ export default class Git {
 	// Gets the commit list in chronological order
 	async getCommitsToPush() {
 		const output = await execCmd(
-			`git log --format=%H --reverse ${ SKIP_PR === false ? `` : `origin/` }${ this.baseBranch }..HEAD`,
+			`git log --format=%H --reverse ${ this.lastCommitSha }..HEAD`,
```

Comment:
base the commits on the difference from the last pushed commit to now. 

This is safe as this reference is not updated until the 
```
		for (const commit of commits) {
			await this.createGithubCommit(commit)
		}
```
that follows.